minecraft_servers/zyneonstudios/manifest.json

```diff
@@ -1,20 +0,0 @@
-{
-    "server_name": "zyneonstudios",
-    "nice_name": "Zyneon Studios",
-    "direct_ip": "zyneonstudios.com",
-    "social": {
-      "web": "https://www.zyneonstudios.com",
-      "twitter": "zyneonstudios",
-      "youtube": "https://www.youtube.com/nerotvlive",
-      "discord": "https://discord.com/invite/DYKArt7A7D"
-    },
-    "discord": {
-      "server_id": 892032464384585768,
-      "rename_to_minecraft_name": false
-    },
-    "brand": {
-      "primary": "#008FE8",
-      "background": "#0A56A5",
-      "text": "#FFFFFF"
-     }
-  }

```

Comment:
Can you delete the whole directory for zyneonstudios?

Reply:
done.

Reply:
thanks!